### ISSUE TITLE
Updating the geometry of curve records

### ISSUE THREAD

**Issue by TomRichardson6** [iho-ohi/S-101-Documentation-and-FC#174]
Within Section 31 of the DCEG the following remark is included;

“When updating the geometry of curve features, compilers must note S-101 clause X.X regarding the requirement for the vector records making up the curve feature to be referenced sequentially. Additionally, for curve features comprising multiple edges, the end node of a vector record must be the same as the start node of the following vector record. It has been reported that some ECDIS reject ENC Updates where the geometry does not conform to these requirements.”

Holger Bothien has kindly provided the following input;

Regarding your question:
The mentioned clause has several problems:
1.	There are no vector records anymore.
2.	There are no edges as well.
3.	I am wondering that this statement is in the update section and not in general.

What is the background.
I assume that the clause should ensure that curve features (features with a geometry of dimension 1, aka lines) should be contiguous. 
Means, there should be no gaps in the geometry.
Since the data model of S-100 is much more complex than the one in S-57 we must consider several cases.
1.	The feature is using exactly one EnGMCurve. S-101 limits the number of segments in the curve to 1, so the curve it continuous and the feature as well.
2.	The feature is using exactly one EnGMCompositeCurve. Since the composite curve has all properties of a curve according to S-100 Part 7 it must be continuous.
This must be achieved by the fact that each component except the fist must start where the previous ends. 
3.	The feature is using multiple curves (Can be EnGMCurve and/or EnGMCompositeCurve). I am not sure if this is considered for S-101 but I could not find anything that limit the number of geometries for a feature. Maybe I have overlooked something. If it is allowed, then it must be ensured that the curves form a continuous geometry. 

Given this I would propose;
1. The removal of this remark from section 31. 

2.  Consider the need for a section at the beginning of the DCEG to cover geometry rules more generally.

3. Consider whether the number of spatial relations should be constrained to 1 in S-101

Your views on these 3 proposals would be welcomed.

**Comment by DavidGrant-NIWC** [comment]
> 3. Consider whether the number of spatial relations should be constrained to 1 in S-101

Yes. Doing otherwise will break the portrayal and introduces issues that are not addressed within S-100. If there is a desire to have multiple spatial relations it should be addressed in S-100 6.0 or later.

**Comment by JeffWootton** [comment]
**S-101PT17:** The changes included in the draft Edition 2.1.0 DCEG, grouping all encoding guidance related to geometry in the DCEG under clause number 2.3, were approved by the meeting. 

Issue to remain open.
